settings.gradle.kts

```diff
@@ -1,2 +1,2 @@
 rootProject.name = "tbdex"
-include("httpclient", "httpserver", "protocol")
+include("httpclient", "httpserver", "protocol", "e2e")
```

Comment:
Does this mean we'll publish `e2e` as a jitpack package @mistermoe ?

Reply:
i _think_ no. chad says https://chat.openai.com/share/d51bd87a-90fa-49c3-bb67-7b4a44b1b11b